source/_components/sensor.nut.markdown

```diff
@@ -94,3 +121,14 @@ sensor:
       - input.voltage
       - battery.runtime
 ```
+
+### {% linkable_title UPS Status - human-readable form %}
+
+An additional virtual sensor type `ups.status.display` is available translating the UPS status value retrived from `ups.status` into a human-readable form.
```

Comment:
retrieved

Reply:
"human-readable version" or "human-readable format"

Reply:
no problem - will fix this now